datahub-frontend/README.md

```diff
@@ -1,11 +1,11 @@
 # DataHub Frontend
 DataHub frontend is a [Play](https://www.playframework.com/) service written in Java. It is served as a mid-tier
-between [DataHub GMS](../gms) which is the backend service and [DataHub Web](../datahub-web).
```

Comment:
Would be better to change the link to reference datahub-web-react/README.md

Reply:
Good suggestion! Updated in [the latest commit](https://github.com/linkedin/datahub/pull/2921/commits/d54f0134d3e93a21a61ecf9cd3727175192c1c55)